index.rst

```diff
@@ -45,3 +48,9 @@ Senior ontwikkelaars zijn: Arjan, Reinout, Jack, Sander, Carsten, Berto (Python)
 Een :doc:`performance budget <performance_budget>` als proxy voor kwaliteit:
 
     Aan het eind van 2016 moet *elke* HTTP request die het pand binnenkomt van buiten binnen **300 ms** een response opgeleverd hebben.
+
+
+3. Tech owner
+-------------
+
+Je bent **volledig vrij** om nieuwe technologie te ontdekken en te introduceren (frameworks, talen, OS, deployment technieken, test strategie, version control, ...) maar zo gauw iets in productie gaat ben je eigenaar (:doc:`tech owner <tech_owner>`) van die technologie en **verantwoordelijk** om minstens 1 mede eigenaar te vinden. Zo lang jij de enige tech owner bent gaat het niet in productie.
```

Comment:
"Volledig vrij" zou ik toch niet zeggen. Iedere leuke taal die ergens handig voor is en ergens wordt uitgetest komt bij N&S vrijwel zeker in productie terecht. En dan valt het opeens onder de tech owner regel en moet er een tweede man gevonden worden. En anders nemen we het toch in productie.

Er gaat volgens mij nauwelijks een remmende werking uit van "tech owner". Gaat niet lukken om iets uit productie te houden en een her-schrijf-actie in python te krijgen want "volgende week moet er een demo aan de klant gegeven worden".

=> Die volledige vrijheid mag wat mij betreft wat minder prominent...